spec/non_conformant/README.md

```diff
@@ -0,0 +1,9 @@
+# `non_conformant/`
+
+## Specs that don't conform to the style guide
```

Comment:
I don't think you need either of these headers—the location of the README is probably enough to indicate what it's referring to.

Reply:
Done.